sor/src/main/java/com/bazaarvoice/emodb/sor/db/test/DeltaClusteringKey.java

```diff
@@ -0,0 +1,61 @@
+package com.bazaarvoice.emodb.sor.db.test;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class DeltaClusteringKey {
+
+    private final UUID _changeId;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_changeId" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00116)